source/app/app.filters/app.orm.filter.ts

```diff
@@ -0,0 +1,118 @@
+import { ArgumentsHost, Catch, ExceptionFilter, HttpStatus, Logger } from '@nestjs/common';
+import { CannotCreateEntityIdMapError, EntityNotFoundError, QueryFailedError, TypeORMError } from 'typeorm';
+
+import { AppException, AppExceptionResponse } from '../app.interface';
+
+@Catch(TypeORMError)
+export class AppOrmFilter implements ExceptionFilter {
```

Comment:
Aqui também o `try...catch` global, porém, para erros do tipo TypeORMError.
Docs: 
https://docs.nestjs.com/exception-filters
https://stackoverflow.com/questions/58993405/how-can-i-handle-typeorm-error-in-nestjs